### PR TITLE
.coafile: Adds PycodestyleBear

### PR DESCRIPTION
**Pull request by lucianzr1**
PycodestyleBear is added to the [autopep8] section of the global .coafile to
run after the PEP8Bear and find problems omitted by the first.

Closes https://github.com/coala/coala/issues/3242

### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [ ] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [ ] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [ ] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [ ] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [ ] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by andreidinu** [comment]
You have to make a single commit. Even if you correct your code, gitmate will show the same errors as at the first commit.

File: coalib/coala_main.py [hunk 1]
```diff
@@ -16,11 +16,6 @@
 from coalib.misc.Constants import configure_logging
 
 
-"""
-do_nothing = lambda *args: True
```
Comment (Nosferatul) [inline coalib/coala_main.py]: Why did you changed this?

**Comment by Nosferatul** [comment]
Rebase and squash your commits

**Comment by Asnelchristian** [comment]
@lucianzr1 would you address the issues pointed out by the others, as for gitmate's ones just run coala in the main directory before pushing

**Comment by Asnelchristian** [comment]
besides you need a rebase

**Pull request closed without merging**